src/Apps/Settings/Routes/SavedSearchAlerts/Components/NewSavedSearchAlertEditForm.tsx

```diff
@@ -144,7 +144,7 @@ const NewSavedSearchAlertEditForm: React.FC<NewSavedSearchAlertEditFormProps> =
           <Box>
             <Join separator={<Spacer y={[4, 6]} />}>
```

Comment:
I thought the edit alert was purposely different, but if we want to adapt it completely - I can add it in the next PR since there are more UI changes coming up that also need clarification first.